settingtypes.txt

```diff
@@ -46,3 +46,12 @@ share_bones_time (Bone share time) int 1200 0
 
 #    Replaces old stairs with new ones. Only required for older worlds.
 enable_stairs_replace_abm (Replace old stairs) bool false
+
+
+#    If enabled, the reason why bones where dropped or not dropped, along
```

Comment:
I suggest rephrasing: `Logs the reason and location of newly placed bones as action.`
otherwise, `where` should be `were` to make sense in this context.